app/views/identities/sites/_site_tabs.html.erb

```diff
@@ -23,7 +23,7 @@
         class: "nav-link #{action_name.to_sym == :stats ? "active" : ""}"
     ) %>
   </li>
-  <% if Flipper[:portal].enabled?(current_user) %>
+  <% if Flipper[:portal].enabled?(FormUser.find(@identity.user_id))%>
```

Comment:
Are you sure we need this change? I think current_user returns a FromUser already...

Reply:
this is purely to make Portal visible to admins as well. not sure if `admins` need to see content of the portal thoiugh @saramic

Reply:
oh right - I get it, we are inheriting the feature flips from the user even if we are navigating as admin - I like it